docs/security/ldap-authentication.md

```diff
@@ -0,0 +1,288 @@
+
+!!! important
+
+    The SASL-based LDAP authentication plugin is a [tech preview](../glossary.md#tech-preview) feature. Before using this plugin in production, we recommend that you test restoring your production databases from physical backups in your environment, and also use an alternative backup method for redundancy.
+
+*Percona Server for MySQL* 8.0.19-10 implements the simple LDAP authentication. The Percona simple LDAP authentication plugin is a free and Open Source implementation of the MySQL Enterprise Simple LDAP authentication plugin.
+
+## Plugin names and file names
+
+The following tables show the plugin names and the file name for simple LDAP authentication and SASL-based LDAP authentication.
+
+=== "Simple LDAP authentication plugin names and library name"
+
+    | Plugin or file | Plugin name or file name |
+    |---|---|
+    | Server-side plugin | authentication_ldap_simple |
+    | client-side plugin | mysql_clear_password |
+    | library file | authentication_ldap_simple.so |
+
+=== "SASL-based LDAP authentication plugin names and library names"
+
+    | Plugin or file | Plugin name or file name |
+    |---|---|
+    | Server-side plugin | authentication_ldap_sasl |
+    | client-side plugin | authentication_ldap_sasl_client |
+    | library files | authentication_ldap_sasl.so <br> authentication_ldap_sasl_client.so  |
+
+## How does the authentication work
+
+The server-side LDAP plugins work only with the specific client-side plugin:
+
+* The `authentication_ldap_simple` plugin, on the server, performs the simple LDAP authentication. The client, using `mysql_clear_password`, connects to the server. The client plugin sends the password to the server as cleartext. For this method, use a secure connection between the client and server.
+
+* The `authentication_ldap_sasl` plugin, on the server, performs the SASL-based LDAP authentication. The client must use the `authentication_ldap_sasl_client` plugin. The method does not send the password to the server in cleartext. The server-side and client-side plugins use Simple Authentication and Security Layer (SASL) to send secure messages within the LDAP protocol.
+
+For either method, the database server rejects the connection if the client user name and the host name do not match a server account.
+
+If a database server LDAP authentication is successful, the LDAP server searches for an entry. The LDAP server matches the user and authenticates using the LDAP password. If the database server account names the LDAP user distinguished name (DN), added by the [`BY clause`](#use-the-by-clause-that-specifies-the-authentication-string), the LDAP server uses that value and the LDAP password provided by the client. This method fails in the following conditions:
+
+* DN and password have incorrect values
+* DN and password are empty and the LDAP server does not permit anonymous connections
```

Comment:
we do not allow anonymous connections (login with an empty password)

Reply:
rewritten.